specification/attributes/column_naming_convention.md

```diff
@@ -24,6 +24,7 @@ Naming convention for columns appearing in billing data.
   * Column IDs SHOULD NOT use acronyms.
   * Column IDs MUST be alphanumeric with no special characters.
   * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
+  * Columns names MUST must not exceed a certain string length. Maximum 64 characters.
```

Comment:
```suggestion
  * Column names SHOULD NOT exceed 64 characters.
```

Reply:
Looks good. This is recommended as many databases have a limit to the length of the column. Ex : mysql allows 64 characters, postgre allows 59 characters.